crypto/ocsp/internal.h

```diff
@@ -35,6 +57,25 @@ struct ocsp_signature_st {
   STACK_OF(X509) *certs;
 };
 
+//   OCSPRequest     ::=     SEQUENCE {
+//       tbsRequest                  TBSRequest,
+//       optionalSignature   [0]     EXPLICIT Signature OPTIONAL }
+//
+struct ocsp_request_st {
+    OCSP_REQINFO tbsRequest;
+    OCSP_SIGNATURE *optionalSignature; /* OPTIONAL */
```

Comment:
Is there something I should check with this `/* OPTIONAL */` that's different than requestExtensions on line 46?